planetiler-core/src/main/java/com/onthegomap/planetiler/archive/WriteableTileArchive.java

```diff
@@ -44,6 +44,10 @@ default void initialize() {}
    */
   default void finish(TileArchiveMetadata tileArchiveMetadata) {}
 
+  default void printStats() {}
+
+  long bytesWritten();
```

Comment:
Thanks for adding this, I could do this in a separate PR as well but there's one remaining place where it gets the size of the archive which could take a very long time if you just wrote tiles for the planet: `Stats.printSummary` - ideally it either uses the same output from this method, or we add a failsafe to `FileUtils.size` that gives up after a certain number of files/time since that output duplicates what got printed earlier we won't miss much.

Reply:
done: https://github.com/onthegomap/planetiler/pull/761/commits/c41b95354f224a112cba09e380ccac9de8aa235b